lib/plugins/aws/deploy/compile/events/apiGateway/lib/apiKeys.js

```diff
@@ -17,8 +17,11 @@ module.exports = {
           throw new this.serverless.classes.Error('API Keys must be strings');
         }
 
+        const apiKeyLogicalId = this.provider.naming
+          .getLogicalApiGatewayApiKeyName(apiKeyNumber);
```

Comment:
I love that we put this under `provider` ❤️ 

Should we namespace the functions such as
`this.provider.naming.events.apigateway.getApiKeyName` or `this.provider.naming.apigateway.getApiKeyName`?